backend/tests/unit/test_load.py

```diff
@@ -12,13 +12,11 @@ def test_data_saved_successfully(test_data_unit_of_work, test_data_repository):
     # Given
     output_dataset_rows = 3
     output_data = [
```

Comment:
It's unclear to me what this testcase is checking?

In unittest we can't test saving, since are mocking repository.
Moreover, if save is indeed tested, we can't use here `queries dto`.